3.Tests/UI/Features/Project/AddAbove.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Project Creation Above
```

Comment:
You should try to organize related tests on the same feature file, we shouldn't really create a separate feature for every scenario.

Reply:
ok